payments/utils.py

```diff
@@ -105,6 +110,51 @@ def _process_payment(order, payment, verbose=0):
         method=payment.get('payment_method', 'fiobank-transfer'),
     )
 
+    if settings.PAYMENT_PROCESS_EMAIL_NOTIFY:
+        event = order.event
+        template = EmailTemplate.objects.get(name='order_update_email')
+        subject = _('Your ticket for {event}.'.format(variable_symbol=order.variable_symbol,
+                                                      event=event.title))
+
+        for ticket in order.ticket_set.all():
+            text_content = template.text_template.format(first_name=ticket.first_name,
+                                                         last_name=ticket.last_name,
+                                                         event=event.title,
```

Comment:
I'm not a big fan of formatting like this one. If we change the variable name from `text_content` for some reason, we will have to also indent all the lines bellow which will create a bunch of changes in git. (Just my opinion, feel free to ignore it.)